src/LtiAssignmentsGradesService.php

```diff
@@ -127,9 +127,8 @@ public function getGrades(LtiLineitem $lineitem = null)
             ServiceRequest::TYPE_GET_GRADES
         );
         $request->setAccept(static::CONTENTTYPE_RESULTCONTAINER);
-        $scores = $this->makeServiceRequest($request);
 
-        return $scores['body'];
+        return $this->getAll($request);
```

Comment:
any potential of this change breaking any other LMS?

Reply:
No, I don't think so - the code for modifying the response body is in [`LtiServiceConnector::getAll()`](https://github.com/packbackbooks/lti-1-3-php-library/blob/master/src/LtiServiceConnector.php#L168), the main difference between that and `makeServiceRequest()` is that `makeServiceRequest()` returns the entire response but `getAll()` returns just the body of the response.

Reply:
nice, I don't have any more concerns then!